.circleci/codeReview.yml

```
@@ -0,0 +1,111 @@
+          command: |
+            echo $GITHUB_PERSONAL_ACCESS_TOKEN | gh auth login --with-token
+            
+      # Run Claude Code for PR review
+      - run:
+          name: Run Claude Code PR Review
+          command: |
+            source $BASH_ENV
+            claude -p "
+            REPO: $CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME
+            PR NUMBER: $CIRCLE_PR_NUMBER
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)